packages/design-system/src/components/Menu/Menu.tsx

```diff
@@ -30,17 +32,18 @@ const Menu = React.forwardRef<React.ReactElement, MenuProps>(
 			<S.Nav
 				isCollapsed={isCollapsed}
 				role="navigation"
-				aria-label="main navigation"
+				aria-label={t('MENU_MAIN_NAVIGATION', 'main')}
 				ref={ref}
 				{...rest}
 			>
 				{hasToggle && (
 					<S.NavButton
-					icon="talend-opener"
+						icon="talend-opener"
 						className={`nav__button ${isCollapsed ? 'nav__button--collapsed' : ''}`}
 						onClick={() => collapse(!isCollapsed)}
+						aria-expanded={!isCollapsed}
 					>
-						{isCollapsed ? 'Uncollapse menu' : 'Collapse menu'}
+						{t('MENu_TOGGLE', 'Toggle menu')}
```

Comment:
I don't know if it's a good way to label it but it seems more logical to use `aria-expanded` here instead of exposing it in a label.